### PR TITLE
chore: exempt stale if milestone is set

### PR DESCRIPTION
**Pull request by qweeah**
This PR adds a workflow change which will avoid marking PRs or issues as stale if any milestone is tagged.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by sajayantony** [comment]
Thanks for the change @qweeah - This also means we should review the milestones regularly so that issues don't get stale in there. Honestly I'm happy that a lot of old issues got bumped and makes the maintainers consider if we actually will ever get to it. 

Future seems like a backlog milestone, should we avoid moving everything to future unless there is plan and/or close them off since there is no immediate plan would be a question for us to agree on.

**Pull request merged**
